doc/docs.md

```diff
@@ -610,20 +610,39 @@ If you do not specify the type explicitly, by default float literals
 will have the type of `f64`.
 
 ### Arrays
-
+#### Basic Array Concepts
```

Comment:
I would just keep "Arrays" for this section.

Reply:
Ah sorry I misread it as a replacement, but it's a new subsection.

It's good.